assigner.py

```diff
@@ -132,23 +133,33 @@ def autocasa(bapi, capi, bcfg, ccfg, dry_run):
                            "using defaults".format(bug.get('assigned_to')))
             delegator_id = casa_status['decidingApprover']['id']
         elif (delegator_id != casa_status['decidingApprover']['id']):
-        ## Set the new assignee if lookup worked
-            res = capi.set_delegator(project_id, delegator_id)
-            logger.info("Setting new assignee/delegator in CASA to {} ({}) for project {}".format(delegator_id,
-                                                                                             bug.get('assigned_to'),
-                                                                                             project_id))
+            ## Set the new assignee if lookup worked
+            if not dry_run:
```

Comment:
Me guesses, you did a dry run and stuff happened, aye?

Reply:
yes, 3 projects now have the assigner correctly set ;P